### PR TITLE
chore(book): update assets for mdbook-admonish

### PR DESCRIPTION
**Pull request by eightfilms**
Fixes CI failure https://github.com/zksecurity/noname/actions/runs/9351865454/job/25738649882

Manually ran the command `mdbook-admonish install` found in the log and committed the changes.

### CODE REVIEW

File: book/book.toml [hunk 1]
```diff
@@ -10,10 +10,10 @@ site-url = "/noname/"
 use-site-url-as-root = true
 curly-quotes = true
 git-repository-url = "https://www.github.com/mimoo/noname"
-additional-css = ["././mdbook-admonish.css"]
+additional-css = ["./mdbook-admonish.css"]
```
Comment (eightfilms) [inline book/book.toml:13]: this was a change I did manually aside from running the command found in the failed job log - we previously seemed to be using a wrong path?

Comment (katat) [reply book/book.toml:13]: Yeah, it looks like a wrong path.

**Comment by katat** [comment]
Thanks!
I am wondering if we have to update these html files? They seem to be generated by the mdbook. @mimoo

**Comment by eightfilms** [comment]
@katat Yeah it was auto-generated. I did a rough check by running `mdbook serve` locally and browsed a few pages to see if anything messed up and it doesn't appear to be, you might want to check too perhaps?

**Comment by katat** [comment]
Ok, then we can exclude these html changes from this PR. I think they will be overwritten by the action builds.

**Comment by eightfilms** [comment]
sure, let me fix this up.

**Comment by eightfilms** [comment]
fixed!

**Comment by katat** [comment]
Thanks so much!

**Pull request merged**